Source/debug.cpp

```diff
@@ -502,6 +595,8 @@ std::vector<DebugCmdItem> DebugCmdList = {
 	{ "cursorcoords", "Toggles showing cursor coords.", "", &DebugCmdShowCursorCoords },
 	{ "grid", "Toggles showing grid.", "", &DebugCmdShowGrid },
 	{ "seedinfo", "Show seed infos for current level.", "", &DebugCmdLevelSeed },
+	{ "mon", "Spawns monster {name}.", "{name}", &DebugCmdSpawnMonsterSingle },
+	{ "mmon", "Spawns up to 8 monsters {name}.", "{name}", &DebugCmdSpawnMonsterMany },

```

Comment:
How about making this a single command with an optional paramter, also rename it to spawn:

```suggestion
	{ "spawn", "Spawns monster [count] {name}.", "{name}", &DebugCmdSpawnMonsterSingle },
```

`spawn 3 skeleton`

`spawn scavenger`

Reply:
if you code that, sure ;)